src/python/sdk/examples/games.py

```diff
@@ -0,0 +1,391 @@
+        ticks_per_lot = random.randint(1 * 10**3, (6 * 10**3) - 1)
+        for (a, b) in clients_pairs:
+            task = asyncio.create_task(
+                setup_pair(
+                    a,
+                    b,
+                    market_id,
+                    (100 * 10**8) // LOT_SIZE,
+                    ticks_per_lot,
+                    initialized,
+                )
+            )
+            tasks.append(task)
+        res = await asyncio.gather(*tasks, return_exceptions=True)
+        initialized = True
+        for result in res:
+            if isinstance(result, Exception):
+                print(f"ERROR: {result}")
+                exit()
+        print(f"Finished round #{i}")
+
+    write_dict_to_file(volume_buffer, "./output.json")
+    print("THE END!")
+
+
+def coin_flip():
+    return random.choice([True, False])  # type: ignore
+
+
+async def setup_pair(
+    client_a: EconiaClient,
+    client_b: EconiaClient,
+    market_id: int,
+    base_lots: int,
+    ticks_per_lot: int,
+    initialized: bool,
+):
+    if not initialized:
+        await client_a.gen_submit_tx_wait(
+            register_market_account(
+                ECONIA_ADDR,
+                COIN_TYPE_EAPT,
+                COIN_TYPE_EUSDC,
+                market_id,
+                0,
+            )
+        )
+        await client_b.gen_submit_tx_wait(
+            register_market_account(
+                ECONIA_ADDR,
+                COIN_TYPE_EAPT,
+                COIN_TYPE_EUSDC,
+                market_id,
+                0,
+            )
+        )
+        print("Initialized account pair...")
+
+    async def run():
+        flip = coin_flip()
+        client_maker = client_a if flip else client_b
+        client_taker = client_b if flip else client_a
+        flip = coin_flip()
+        from_type = COIN_TYPE_EAPT if flip else COIN_TYPE_EUSDC
+        to_type = COIN_TYPE_EUSDC if flip else COIN_TYPE_EAPT
+        base_lots_remaining = base_lots
+        await client_a.gen_submit_tx_wait(
+            deposit_from_coinstore(ECONIA_ADDR, COIN_TYPE_EAPT, market_id, 0, 100 * 10**8)
+        )
+        await client_a.gen_submit_tx_wait(
+            deposit_from_coinstore(
+                ECONIA_ADDR, COIN_TYPE_EUSDC, market_id, 0, 600 * 10**6
+            )
+        )
+        await client_b.gen_submit_tx_wait(
+            deposit_from_coinstore(ECONIA_ADDR, COIN_TYPE_EAPT, market_id, 0, 100 * 10**8)
+        )
+        await client_b.gen_submit_tx_wait(
+            deposit_from_coinstore(
+                ECONIA_ADDR, COIN_TYPE_EUSDC, market_id, 0, 600 * 10**6
+            )
+        )
+
+        orders = 0
+        while base_lots_remaining > MIN_SIZE:
+            base_lots_size = random.randint(MIN_SIZE, base_lots_remaining)
+            match_price = (
+                ticks_per_lot if from_type == COIN_TYPE_EUSDC else ticks_per_lot + 1
+            )
+            volume = base_lots_size * match_price * TICK_SIZE
```

Comment:
@CRBl69 This is where I'm calculating expected volume

Reply:
@elliottdehn is this resolved?